frontend/src/components/posts/buttons/create-post-button.tsx

```diff
@@ -3,6 +3,10 @@
 import Link from "next/link";
 import styles from "./post-buttons.module.css"
 import { gotoCreatePostPage } from "@/utils/url-segments";
+=======
+import styles from "./post-buttons.module.css"
+import { gotoCreatePostPage } from "@/utils/posts/url-segments";
+>>>>>>> b905c44 (Update create-post-button.tsx)
```

Comment:
I think you forgot to resolve the merge conflicts in create-post-button.tsx. The new lines
```
======
...
>>>>>>
```
should be deleted.